Android apk build

I am not Android developer, so maybe stupid question, but I am not able to compile the application with this error:

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > Project :app declares a dependency from configuration 'compile' to configuration 'default' which is not declared in the descriptor for project :bluetoothspp.

Please could you help me?

--- a/Android/ChorusRFLaptimer/app/build.gradle
+++ b/Android/ChorusRFLaptimer/app/build.gradle
@@ -29,7 +29,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.1.0'
     compile 'com.android.support:design:25.1.0'
-    compile project(':bluetoothspp')
+    compile project(':bluetoothSPP')
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
 }

--- a/Android/ChorusRFLaptimer/settings.gradle
+++ b/Android/ChorusRFLaptimer/settings.gradle
@@ -1,1 +1,1 @@
-include ':app', ':bluetoothspp'
+include ':app', ':bluetoothSPP'

